CONTRIBUTING.md

```diff
@@ -16,4 +16,4 @@ If your bug fix does not follow our style and contribution guidelines, you will
 
 ### GitFlow and the Release Cycle
 
-Please read about  [GitFlow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) to understand how Flambé handles git and GitHub policies. We aim to release patches at least once a week, but currently follow no other regular cycles or sprint schedules.
+Please read about [GitFlow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) to understand how Flambé handles git and GitHub policies. We aim to release patches at least once a week, but currently follow no other regular cycles or sprint schedules.
```

Comment:
This actually needs to go away, we don't follow gitflow anymore. It was overkill :)